<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanThinkDlg</class>
 <widget class="QDialog" name="PlanThinkDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>想定方案编制</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> 	font-family:Microsoft Yahei;
    font-size:13pt;
	</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>训练模式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>训练模式</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>考核模式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">image: url(:/EControlSys/res/line.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>基础信息</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">image: url(:/EControlSys/res/line.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>235</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok">
      <property name="text">
       <string>生成方案</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>618</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>科目：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>任务名称：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入任务名称</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>人员：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_btn">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>训练编号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>自动生成</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>预计开始时间：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>0</height>
         </size>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>yyyy/M/d H:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>训练目的/目标：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入训练目的</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EControlSys.qrc"/>
 </resources>
 <connections/>
</ui>
